### PR TITLE
Add navigation links cog 64

### PR DESCRIPTION
**Pull request by kelfink**
Add navigation links on user list and user detail  pages, pointing at the list and the overall dashboard pages.

### CODE REVIEW

File: app/javascript/views/usersList/UsersList.jsx [hunk 1]
```diff
@@ -1,15 +1,35 @@
...
+              text="Dashboard"
+              href={dashboardUrl}
+              clickHandler={dashboardClickHandler}
+            />
+            &nbsp;&gt;&nbsp;
+            <Link
+              text="Manage Users"
+              href={userListUrl}
+              clickHandler={userListClickHandler}
+            />
```
Comment (Sushmareddy-OSI) [inline app/javascript/views/usersList/UsersList.jsx]: As per the design, on this page we need link only for dashboardUrl.

screen shot 2018-05-15 at 2 01 57 pm

File: spec/acceptance_helper.rb [hunk 2]
```diff
@@ -16,6 +16,6 @@
 
   config.default_max_wait_time = 10
   config.default_driver = :selenium
-  config.app_host = ENV.fetch('CASE_WEB_BASE_URL', 'http://localhost:3000')
+  config.app_host = ENV.fetch('C_WEB_BASE_URL', 'http://localhost:3000')
```
Comment (RamuKammagani-osi) [inline spec/acceptance_helper.rb]: Why is it C_WEB_BASE_URL? isn't it COUNTY_ADMIN_WEB_BASE_URL??

File: app/javascript/views/usersList/UsersList.jsx (hunk 1 above)
Comment (RamuKammagani-osi) [reply app/javascript/views/usersList/UsersList.jsx]: +1

File: spec/acceptance_helper.rb (hunk 2 above)
Comment (kelfink) [reply spec/acceptance_helper.rb]: yes!  whoops

**Comment by RamuKammagani-osi** [comment]
retest this please

**Pull request merged**